packages/site-client/tsconfig.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "rootDir": "./src",
     "noEmit": true,
-    "tsBuildInfoFile": "tsconfig.tsbuildinfo"
+    "tsBuildInfoFile": "tsconfig.tsbuildinfo",
+    "useDefineForClassFields": false
```

Comment:
This doesn't actually do anything since we don't emit from tsc, right? Or does esbuild automatically read this file?

Reply:
esbuild reads it now without the flags I removed.